buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -550,7 +551,13 @@ def set_node(job_type) {
         if (!NODE) {
             error("Missing ${job_type} NODE!")
         }
+    }
 
+    if (NODE.contains('docker')) {
```

Comment:
I'm not sure why this `if`. Shouldn't we only care if the variable file has `docker_image` for the spec? I think this means the node name must contain `docker`, which is not always true.

Reply:
Reworked and removed.